clmm/theory/parent_class.py

```diff
@@ -906,3 +1094,22 @@ def convert_mass_concentration(self, z_cl, massdef=None, delta_mdef=None,
 
         return self._convert_mass_concentration(z_cl, massdef, delta_mdef,
                                                 halo_profile_model, alpha)
+
+    def _validate_z_src(self, loc_dict):
+        r"""Validation for z_src
+
+        Parameters
+        ----------
+        locals_dict: dict
+            Should be the call locals()
+        """
+        if loc_dic['z_src_info']=='discrete':
```

Comment:
`loc_dic` --> `loc_dict`. The same applies to the two elif's below and `beta_info`.